packages/workflow/src/interfaces.ts

```diff
@@ -538,6 +538,11 @@ export type Handler<
  */
 export type DefaultSignalHandler = (signalName: string, ...args: unknown[]) => void | Promise<void>;
 
+/**
+ * A handler function accepting query calls for non-registered query names.
+ */
+export type DefaultQueryHandler = (queryName: string, ...args: any[]) => any;
```

Comment:
nit: That should be `unknown` rather than `any` (both occurrences).

Reply:
Done.